### DISCUSSION TITLE
Autocomplete Enterkey

### DISCUSSION THREAD

**Discussion by bytrus** [vikramlearning/blazorbootstrap#408] [Q&A]
Hi, 
im using blazorbootstrap in a wasm Project. 

If i choose a Item in a Autocomplete component with Enterkey the Form want's to submit. Is there a way to prevent this? (eg stoppropagination etc..)

many thanks
bytrus

**Comment by gvreddy04** [comment]
@bytrus Thanks for trying **BlazorBootstrap** components. Can you please share the sample code to reproduce the issue with minimal steps? It would be helpful if you could also attach a screen recording.

**Reply by bytrus** [reply]
Hi,
thanks for reply. I worte a little example where you can track what i said. Whenever you choose a title item via cursor down + enter, it will submit the form. See your console.
got to Index.razor
thanks for your help
cheers.

**Comment by bytrus** [comment]
[TypeAhead.zip](https://github.com/vikramlearning/blazorbootstrap/files/13039856/TypeAhead.zip)
